File: quine.cs
using System;
public class Quine
{
  public static void Main()
  {
    string s = "using System;{0}public class Quine{0}{1}{0}{2}public static void Main(){0}{2}{1}{0}{2}{2}string s = {3}{4}{3};{0}{2}{2}Console.WriteLine(String.Format(s,(char)10,(char)123,{3}  {3},(char)34,s,(char)125));{0}{2}{5}{0}{5}";
    Console.WriteLine(String.Format(s,(char)10,(char)123,"  ",(char)34,s,(char)125));
  }
}

File: quine.vb
Imports System
Module Main
  Sub Main()
    Dim s As String = "Imports System{0}Module Main{0}  Sub Main(){0}    Dim s As String = {1}{2}{1}{0}    Console.WriteLine(s,Chr(10),Chr(34),s){0}  End Sub{0}End Module"
    Console.WriteLine(s,Chr(10),Chr(34),s)
  End Sub
End Module
